src/include/reconcile.h

```diff
@@ -256,12 +256,15 @@ struct __wt_reconcile {
     WT_CURSOR_BTREE update_modify_cbt;
 
     /*
-     * Variables to track reconciled pages containing cells with time window values and prepared
+     * WT_REC_CELL_STATS--
```

Comment:
I'm not averse to putting these into a structure, but I don't think it's necessary any more. I also don't think you need to give the structure a type (if you do, this doesn't look quite right).

Reply:
Fixed